### PR TITLE
LTV: Use alias as fallback to original series name

### PR DESCRIPTION
**Pull request by fernandog**
Fixes an old issue when it doesn't find subtitles because show in provider uses a different name

Examples:
`TURN` and `Turn: Washington's Spies`
`DC’s Legends of Tomorrow` and `Legends of Tomorrow`
`Marvels Agents of SHIELD` vs `Agents of SHIELD`
`Marvel's Jessica Jones` vs `Jessica Jones`
*Most Marvel's shows

It will also work for releases with translated releases like `La Lista Negra` for `The Blacklist`

@ratoaq2 @duramato

PR is in sync with upstream

### CODE REVIEW

**Comment by fernandog** [comment]
@ratoaq2 waiting your approval

File: medusa/subtitle_providers/legendastv.py [hunk 1]
```diff
@@ -122,7 +122,8 @@ def get_matches(self, video, hearing_impaired=False):
         # episode
         if isinstance(video, Episode) and self.type == 'episode':
             # series
-            if video.series and sanitize(self.title) == sanitize(video.series):
+            if video.series and (sanitize(video.series) in (
```
Comment (ratoaq2) [inline medusa/subtitle_providers/legendastv.py]: This is wrong. It's inverted self e video

File: medusa/subtitle_providers/legendastv.py [hunk 2]
```diff
@@ -136,7 +137,8 @@ def get_matches(self, video, hearing_impaired=False):
         # movie
         elif isinstance(video, Movie) and self.type == 'movie':
             # title
-            if video.title and sanitize(self.title) == sanitize(video.title):
+            if video.title and (sanitize(video.title) in (
```
Comment (ratoaq2) [inline medusa/subtitle_providers/legendastv.py]: Same here

**Comment by fernandog** [comment]
@ratoaq2 done. don't know why I did that mistake while cherry-picking your commit

**Pull request merged**